packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -153,6 +172,7 @@ export function DatePicker({
         ]}
         highlightDates={highlightDates}
         onMonthChange={onMonthChange}
+        locale={effectiveLocale}
```

Comment:
is this working for you?

I tried throwing in a few different locales to see what DatePicker even does with a simple string and all I got was a bunch of console errors 😭

Reply:
👀  works for me - showing Monday as it should with `enGB`

<img width="1435" alt="Screenshot 2025-04-21 at 8 50 51 PM" src="https://github.com/user-attachments/assets/fd8884b8-4238-45ee-848d-dca09f081908" />

Reply:
Resolving this since we scrapped `locale`